docs/guide/deployment.md

```diff
@@ -133,6 +133,15 @@ Commit both the `.gitlab-ci.yml` and `vue.config.js` files before pushing to you
 
 Also checkout [vue-cli-plugin-netlify-lambda](https://github.com/netlify/vue-cli-plugin-netlify-lambda).
 
+In order to receive direct hits using `History Pushstate mode` on Vue Router, you need to create a file called `_redirects` under `/public` with the following content:
```

Comment:
```suggestion
In order to receive direct hits using `history mode` on Vue Router, you need to create a file called `_redirects` under `/public` with the following content:
```
It's commonly referred to as "`history` mode" in our docs etc, so we should keep it that way.